lib/evaluateConstantSum.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const clone = require('clone');
+
+const ConstantFraction = require('./ConstantFraction');
+const evaluateArithmetic = require('./evaluateArithmetic');
+const MathChangeTypes = require('./MathChangeTypes');
+const NodeCreator = require('./NodeCreator');
+const NodeStatus = require('./NodeStatus');
+const NodeType = require('./NodeType');
+
+// Evaluates a sum of constant numbers and integer fractions to a single
+// constant number or integer fraction. e.g. e.g. 2/3 + 5 + 5/2 => 49/6
+// Returns a NodeStatus object.
+function evaluateConstantSum(node) {
+  if (NodeType.isParenthesis(node)) {
+    node = node.content;
+  }
+  if (!NodeType.isOperator(node) || node.op !== '+') {
+    return NodeStatus.noChange(node);
+  }
+
+  // functions needed to evalute the sum
+  const summingFunctions = [
+    evaluateArithmetic,
+    ConstantFraction.addConstantFractions,
+    ConstantFraction.addConstantAndFraction,
+  ];
+  for (let i = 0; i < summingFunctions.length; i++) {
+    const status = summingFunctions[i](node);
+    if (status.hasChanged()) {
+      if (NodeType.isConstantOrConstantFraction(status.newNode)) {
+        return status;
+      }
+    }
+  }
+
+  let oldNode = clone(node, false);
+  const subSteps = [];
+
+  // STEP 1: group fractions and constants separately
+  let status = groupConstantsAndFractions(oldNode);
```

Comment:
I use constant in most places in the code to refer to constant nodes, ie a single node with any number value (whole number or decimal number)

should I add this to the readme? should we rename things?